components/learn/modules/ROOT/pages/developing-smart-contracts.adoc

```diff
@@ -51,38 +51,35 @@ Init successful, sweet!
 ```
 --
 
-[.buidler]
+[.hardhat]
 --
-To get started with Buidler we will install it in our xref:setting-up-a-node-project.adoc#creating-a-project[project directory].
+To get started with Hardhat we will install it in our xref:setting-up-a-node-project.adoc#creating-a-project[project directory].
 
 ```console
-$ npm install --save-dev @nomiclabs/buidler
+$ npm install --save-dev hardhat
 ```
 
-Once installed, we can create an empty Buidler config file (`buidler.config.js`) in our project directory.
+Once installed, we can run ``npx hardhat``. This will create an empty Hardhat project in our project directory.
 
 ```console
-$ npx buidler
-888               d8b      888 888
-888               Y8P      888 888
-888                        888 888
-88888b.  888  888 888  .d88888 888  .d88b.  888d888
-888 "88b 888  888 888 d88" 888 888 d8P  Y8b 888P"
-888  888 888  888 888 888  888 888 88888888 888
-888 d88P Y88b 888 888 Y88b 888 888 Y8b.     888
-88888P"   "Y88888 888  "Y88888 888  "Y8888  888
-
-Welcome to Buidler v1.4.4‍
-
-✔ What do you want to do? · Create an empty buidler.config.js
-Config file created
+$ npx hardhat
+888    888                      888 888               888
+888    888                      888 888               888
+888    888                      888 888               888
+8888888888  8888b.  888d888 .d88888 88888b.   8888b.  888888
+888    888     "88b 888P"  d88" 888 888 "88b     "88b 888
+888    888 .d888888 888    888  888 888  888 .d888888 888
+888    888 888  888 888    Y88b 888 888  888 888  888 Y88b.
+888    888 "Y888888 888     "Y88888 888  888 "Y888888  "Y888
+
+Welcome to Hardhat v2.0.0
+
+? What do you want to do? …
+❯ Create a sample project
+  Create an empty hardhat.config.js
+  Quit
 ```
 
-We can then create a `contracts` directory.
```

Comment:
Any reason to remove this?  For community coming from Truffle they will have to create a `contracts` directory.